addons/sourcemod/scripting/vsh/bosses/boss_pyrocar.sp

```diff
@@ -196,6 +198,10 @@ methodmap CPyroCar < SaxtonHaleBase
 		//Disable self-damage from bomb rage ability
 		if (this.iClient == attacker && strcmp(sWeaponClassName, "tf_generic_bomb") == 0)
 			return Plugin_Stop;
+			
+		float flHealingRate = 1.0;
+		if (TF2_IsPlayerInCondition(this.iClient, TFCond_Milked) && this.iClient != attacker && TF2_FindAttribute(attacker, ATTRIB_LESSHEALING, flHealingRate))
+			Client_AddHealth(attacker, RoundToNearest(damage - flHealingRate/damage));
```

Comment:
Mad milk should already use `TakeHealth`, taking into account with less healing attrib. Its just `Client_AddHealth` not taking into account with it until this commit